alembic/versions/c3da223e60d8_add_has_backup_field.py

```diff
@@ -0,0 +1,57 @@
+"""add has_backup field
+
+Revision ID: c3da223e60d8
+Revises: d0aa961845b9
+Create Date: 2023-09-22 11:10:14.616339
+
+"""
+import sqlalchemy as sa
+from sqlalchemy.dialects import mysql
+from sqlalchemy.orm import sessionmaker
+
+from alembic import op
+from cg.store.models import Flowcell
+
+# revision identifiers, used by Alembic.
+revision = "c3da223e60d8"
+down_revision = "d0aa961845b9"
+branch_labels = None
+depends_on = None
+
+Session = sessionmaker()
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    bind = op.get_bind()
+    session = Session(bind=bind)
+
+    op.alter_column(
+        table_name="application",
+        column_name="min_sequencing_depth",
+        existing_type=mysql.INTEGER(),
+        nullable=False,
+    )
```

Comment:
This automatically generated alteration was unexpected. I see that in #1904 this field was modified to be non-nullable but the database doesn't seem to have been modified.

Reply:
Does this need to be modified in the SQL schema?

Reply:
I am considering removing it. It is out of the scope of this PR but is an inconsistency that has to be addressed at some point.